### ISSUE TITLE
JS Error on upon render of div with soho-busyindicator directive with scroll bar

### ISSUE THREAD

**Issue by ayoayco** [infor-design/enterprise-ng#684]
**Describe the bug**
We are encountering this error when rendering a soho-busyindicator that has a vertical scroll bar.

**To Reproduce**
1. Add a div with soho-busyindicator directive on a modal
2. Set the height of a modal
2. Summon the modal in a browser window small enough to get a vertical scrollbar in the modal
3. Check JS console
4. See error:
```
zone.js:199 Uncaught TypeError: Cannot read property 'css' of undefined
    at BusyIndicator.addScrollParent (sohoxi.js:22028)
    at sohoxi.js:21872
    at ZoneDelegate.push../node_modules/zone.js/dist/zone.js.ZoneDelegate.invokeTask (zone.js:423)
    at Zone.push../node_modules/zone.js/dist/zone.js.Zone.runTask (zone.js:195)
    at push../node_modules/zone.js/dist/zone.js.ZoneTask.invokeTask (zone.js:498)
    at ZoneTask.invoke (zone.js:487)
    at timer (zone.js:2281)
```

**Expected behavior**
No error

**Version**
- ids-enterprise-ng: 4.20.1

**Screenshots**

**Platform**
 - Device (if applicable) Laptop HP
 - OS Version: Windows 10
 - Browser Name: Chrome
 - Browser Version: Version 78.0.3904.108 (Official Build) (64-bit)

**Comment by tmcconechy** [comment]
This may be an EP bug but i cannot reproduce this try your steps on this page:

http://latest-enterprise.demo.design.infor.com/components/busyindicator/test-inside-modal.html
1. show indicator
2. resize to smaller size

Perhaps this error is now fixed as this is version 4.23

**Comment by tmcconechy** [comment]
I was not able to reproduce this issue. Assume that it was fixed on a newer version.
